react/new/src/constants/nhlTeams.ts

```diff
@@ -0,0 +1,34 @@
+export const NHL_TEAM_NAMES: Record<string, string> = {
```

Comment:
Not that it looks like you need something like this, but this is the very dumb way I handle multiple names/abbrevs for Wes: https://github.com/sprx97/WesBot/blob/master/Shared.py

Reply:
We could do that pretty easily here:

` 'Anaheim Ducks': ['ANA', 'ANAH']` for example, then implement a method that gets the team name based on either of those abbreviations, or we could just add multiple keys for the same team, but I think the array approach is cleaner.

Reply:
Yeah -- it's not necessary here because it's all internal, but I had to come up with a solution for N different ways people would try to /score or /ot a team name.